.ci/build.sh

```diff
@@ -9,21 +9,23 @@ mv cabal.project.local cabal.project.local.disabled
 cabal new-build --dry-run all > /dev/null || (echo Maybe the index-state should be updated?; false)
 mv cabal.project.local.disabled cabal.project.local
 
-cabal new-build all
+if [ "$RUN_BUILD_ALL" = "yes" ]; then
```

Comment:
Use `[[` over `[`, it's safer and we're already using `bash` instead of `sh`

Reply:
ShellCheck betrayed me.

Reply:
`[[` is non-POSIX, that might be why. But we're explicitly using bash anyway so I'd rather see `[[` because it has fewer footguns